core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -556,6 +556,7 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationDiffService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationEditorAdvancedFeaturesService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationSaveService.js"></script>
```

Comment:
One request: do you think you could make new html files for the individual modals, too, into files similar to help_modal_directive.html? That will move us towards the goal of "one file per thing", and I think it would also make it easier to conceptually treat each modal as its own thing rather than as part of the large exploration_editor.html file.

Thanks!